supplementary_style_guide/introduction/about-guide.adoc

```diff
@@ -1,21 +1,28 @@
 = About this guide
 
-This guide provides style guidelines and preferred term usage for link:https://www.redhat.com/[Red Hat] product and cross-product solution documentation. It is intended to be used as a supplement to the _IBM Style_ guide, which is the primary source of guidance for Red Hat product documentation.
+The style and language guidance in this guide overrides or supplements some guidance provided by _IBM Style_, which is the primary source of style guidance for link:https://www.redhat.com/[Red Hat] product and cross-product solution documentation. 
 
-The Red Hat Customer Content Services team created this guide as part of its responsibility for producing product documentation for Red Hat customers. Upstream communities who want to align more closely with the standards used by Red Hat product documentation can also use this guide.
+The Red{nbsp}Hat Customer Content Services team has created this guide as part of its responsibility for producing product documentation for Red{nbsp}Hat customers. Upstream communities who want to align more closely with the standards used by Red{nbsp}Hat product documentation can also use this guide.
 
 [NOTE]
 ====
-Other Red Hat technical documentation, including Red Hat training and exam content by Global Learning Services (GLS), follows the link:https://stylepedia.net/[_Red Hat Technical Writing Style Guide_] instead of the _Red Hat supplementary style guide for product documentation_.
+Other Red Hat technical documentation, including Red Hat training and exam content by Global Learning Services (GLS), follows the link:https://stylepedia.net/[_Red Hat Technical Writing Style Guide_] instead of the _Red{nbsp}Hat supplementary style guide for product documentation_.
 ====
 
-== How to use this guide
+== Using style guides for Red{nbsp}Hat product documentation
 
-When looking for style guidance, reference this guide first, because it overrides certain guidance from the _IBM Style_ guide. If you must deviate from the guidance in either guide, notify the style council by opening an link:https://github.com/redhat-documentation/doc-style/issues[issue]. This way, the deviation can be discussed by the style council and included in this guide.
+Red{nbsp}Hat product documentation adheres to the style guidance provided in _IBM Style_, the _Red{nbsp}Hat supplementary style guide for product documentation_, and style guides specific to Red{nbsp}Hat products, such as Red{nbsp}Hat Enterprise Linux or Red{nbsp}Hat Openshift Container Platform.
 
-== Related Red Hat guides
+When seeking style guidance, consult the style guide specific to your product documentation project first, if applicable, then the _Red{nbsp}Hat supplementary style guide for product documentation_, and finally _IBM Style_.
+The following image illustrates the relationship between the different style guides in Red{nbsp}Hat product documentation:
```

Comment:
Instead of just "relationship", would it be more accurate to say that it demonstrates the order?

```suggestion
The following image illustrates the order to consult the different style guides in Red{nbsp}Hat product documentation:
```

Reply:
Changed to "hierarchical relationship" because it's not just about the order to use when consulting them.